SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc

```diff
@@ -0,0 +1,71 @@
+The OBX-18 field shall contain the equipment (or device) identifier on the MDS level and/or the measurement module identifier of the equipment on the VMD level.
+
+The table <<ref_tbl_dec_obx18_mds_mapping>> defines the mapping of the SDC MDS meta data to the data fields of the HL7 data type *EI* used in the OBX-18 field.

```

Comment:
We should introduce appropriate glossary items for MDS and VMD and remove "SDC".

Reply:
I added SDC MDIB instead of SDC since I want to set the context on the MDIB data of the MDS / VMD element in the MDIB and not in the HL7 context which is using the same terms.